app/pandas.ipynb

```diff
@@ -208,18 +476,96 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
    "source": [
```

Comment:
The moving average calculation is implemented here, but it seems to be using a window of 2 instead of 5. Ensure to use a window of 5 as per the task requirements.